lib/rubocop/cop/lint/shadowing_outer_local_variable.rb

```diff
@@ -34,12 +37,17 @@ module Lint
       class ShadowingOuterLocalVariable < Base
         MSG = 'Shadowing outer local variable - `%<variable>s`.'
 
+        def_node_matcher :ractor_block?, <<~PATTERN
+          (block (send (const nil? :Ractor) :new ...) ...)
```

Comment:
Could you use `#global_const?(:Ractor)` instead?

Reply:
Thanks. For some reason `global_const` method is not found. Am I missing something?

Reply:
Oh, right, it's a method of `Node` only, but since the receiver could be `nil` you'd have to write `[!nil? global_const?(:Ractor)]`. Let's merge it as is and I'll figure something out.

Reply:
@marcandre Changed back to `const nil?`

Reply:
Cool, thanks. I created https://github.com/rubocop-hq/rubocop-ast/issues/160